plugins/storage/volume/datera/src/main/java/org/apache/cloudstack/storage/datastore/provider/DateraHostListener.java

```diff
@@ -18,6 +18,23 @@
  */
 package org.apache.cloudstack.storage.datastore.provider;
 
+import java.io.UnsupportedEncodingException;
```

Comment:
@anuragaw - Just checking - why was this file modified? I don't expect it has anything to do with this feature (I see lines placing has changed).

Reply:
This seems like an IDE auto-refactor induced change (accidental). I'll revert this change from the PR.